lambdas/scripts/build_zip.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+set -e
+
+# Make sure "*" expands to an empty list rather than a literal "*" if there are no matches.
+shopt -s nullglob
+
+error() {
+    echo $@ 2>&1
+    exit 1
+}
+
+[ -f "/.dockerenv" ] || error "This should only run inside of quiltdata/lambda container."
+
+mkdir out
+cd out
+
+pip3 install -U pip setuptools
+
+# remove pre-installed lambda packages from requirements.txt
+preinstalled=$(pip freeze | cut -d '=' -f 1)
+preinstalled_regex=${preinstalled//$'\n'/|}
+grep -v -E -i "^(${preinstalled_regex})==" /lambda/function/requirements.txt > filtered_requirements.txt || true
```

Comment:
looks outdated

Reply:
Why? This was intentional: the lambda runtime already contains e.g. boto3, so we don't want to try to install it ourselves.

Reply:
https://github.com/quiltdata/deployment/pull/1692

Reply:
ah, ok, updated it